testdata/modules/openconfig-withlist.yang

```diff
@@ -40,7 +40,9 @@ module openconfig-withlist {
 
       container b {
         list multi-key {
-          key "key1 key2";
+          key "key1
```

Comment:
could also have a specific test for multiple spaces here; but since the new implementation treats both cases the same, probably not needed.

Reply:
I added spaces as well to the irregular spacing integration test